feat: partner params key encoding errors (#169)
smileidentity/ios: CHANGELOG.md

# Release Notes

## 10.1.2

#### Fixed

> * partner params duplicate keys error from user_id, job_id and job_type

Reviewer: nit
```suggestion
* Ignore `user_id`, `job_id`, and `job_type` fields in `extraPartnerParams`
```
